### PR TITLE
Add explicit_start flag

### PR DESCRIPTION
**Pull request by dudicoco**

### CODE REVIEW

**Comment by dudicoco** [comment]
@linuxdaemon

File: pre_commit_hooks/yamlfmt [hunk 1]
```diff
@@ -63,6 +63,12 @@ class Cli:
             action="store_true",
             help="whether to keep existing string quoting"
             )
+        parser.add_argument(
+            "-e",
+            "--explicit-start",
+            action="store_true",
+            help="whether to add an explicit document start"
+            )
```
Comment (linuxdaemon) [inline pre_commit_hooks/yamlfmt:71]:
```suggestion
        parser.add_argument(
            "-e",
            "--explicit-start",
            action="store_true",
            default=True,
            help="whether to add an explicit document start"
            )
```
To keep the old behavior.

File: pre_commit_hooks/yamlfmt [hunk 2]
```diff
@@ -138,6 +144,7 @@ if __name__ == "__main__":
         colons=ARGS.colons,
         width=ARGS.width,
         preserve_quotes=ARGS.preserve_quotes
```
Comment (linuxdaemon) [inline pre_commit_hooks/yamlfmt:146]:
```suggestion
        preserve_quotes=ARGS.preserve_quotes,
```

File: pre_commit_hooks/yamlfmt (hunk 1 above)
Comment (dudicoco) [reply pre_commit_hooks/yamlfmt:71]: @linuxdaemon isn't handled in `yaml.explicit_start = kwargs.get("explicit_start", True)`?

Comment (linuxdaemon) [reply pre_commit_hooks/yamlfmt:71]: no, the get only applies if the kwarg isn't passed to the function, not if the argument isn't set.

**Comment by dudicoco** [comment]
@linuxdaemon when can this be merged and released? Thanks!

**Comment by linuxdaemon** [comment]
@dudicoco I don't control this project, I just contribute

**Comment by dudicoco** [comment]
@jumanjiman can we merge this, please?

**Comment by jumanjiman** [comment]
@dudicoco @linuxdaemon 

thank y'all!

i also bump the tag to 0.0.12

**Comment by dudicoco** [comment]
Thanks @jumanjiman!

**Comment by dudicoco** [comment]
@linuxdaemon @jumanjiman it seems that with this implementation it's not actually possible to set `explicit_start` to `False`. Perhaps it's better to add an `implicit_start` flag instead?

**Pull request merged**